src/azure-cli/azure/cli/command_modules/servicefabric/operations/managed_clusters.py

```diff
@@ -167,6 +167,12 @@ def add_client_cert(cmd,
         if not cluster.public_i_pv6_prefix_id:
             cluster.public_i_pv6_prefix_id = None
 
+        if not cluster.public_ip_prefix_id:
```

Comment:
was this intentional? it looks duplicated from above

Reply:
two different properties

Reply:
Duplicated from above. line 164 to 168 is the same as 170 to 174